browser/profiles/brave_profile_impl.cc

```diff
@@ -20,6 +25,17 @@ BraveProfileImpl::BraveProfileImpl(
     scoped_refptr<base::SequencedTaskRunner> io_task_runner)
     : ProfileImpl(path, delegate, create_mode, creation_time, io_task_runner),
       weak_ptr_factory_(this) {
+#if !defined(OS_ANDROID)
+  const PrefService::Preference* pref =
+      GetPrefs()->FindPreference(kShieldsAdvancedViewEnabled);
+  if (!pref->HasUserSetting()) {
+    // advanced view is defaulted to true for EXISTING users; false for new.
+    // preference needs to be explicitly set to hold its value
+    bool default_value = first_run::IsChromeFirstRun() == false;
```

Comment:
nit: This looks strange to me :)
How about using `const bool default_value = !first_run::IsChromeFirstRun()`?

Reply:
Good call- fixed! 😄